tensorflow_probability/python/stats/kendalls_tau_test.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Tests Kendall's Tau metric."""
+
+import random
+
+import numpy as np
+from scipy import stats
+
+import tensorflow.compat.v2 as tf
+import tensorflow_probability as tfp
+from tensorflow_probability.python.internal import test_util
+
+@test_util.test_all_tf_execution_regimes
+class KendallsTauTest(test_util.TestCase):
+
+  def test_kendall_tau(self):
+    x1 = [12, 2, 1, 12, 2]
+    x2 = [1, 4, 7, 1, 0]
+    expected = stats.kendalltau(x1, x2)[0]
+    res = tfp.stats.kendalls_tau(tf.constant(x1, tf.float32),
+                                 tf.constant(x2, tf.float32))
+    np.testing.assert_allclose(expected, res.numpy(), atol=1e-5)
```

Comment:
Also, the preferred approach to getting a Tensor's value in a test is `self.evaluate(res)` (which works in both eager and graph-mode testing) rather than `res.numpy()` (which only works in eager mode).

Reply:
Done.